docs: Document student answers classification (#421)
ocaml-sf/learn-ocaml: docs/howto-classify-students-answers.md

How to classify students answers
================================

> # Context

Reviewer: ```suggestion
## Context
```

If you are a teacher with a large number of students, you probably
want to get an idea of your students' typical answers to a given
question without having to look at each of them individually.

LearnOCaml provides an (experimental) feature to *automatically

> classify students answers*. Provided a given identifier, LearnOCaml

Reviewer: `Provided` sounds not that natural for me; maybe just `From`?
```suggestion
classify students answers*. From a given identifier, LearnOCaml
```

computes the disimilarities between students' answers and regroup

> them in different cluster, so that the teacher can only look at

Reviewer: ```suggestion
them in different clusters, so that the teacher can only look at
```

> one representant of each cluster to get a quite complete feeling

Reviewer: ```suggestion
one representant of each cluster to get a fairly comprehensive overview
```
(maybe?)

> of students' ways to approach to the assignment.

Reviewer: ```suggestion
of students' ways to approach the assignment.
```

> # How-to

Reviewer: ```suggestion
## How-to
```

> To analyze students' definitions of function `foo` in exercise `x`.

Reviewer: ```suggestion
Let's see how to analyze students' definitions of function `foo` in exercise `x`.
```

Follow these steps:

> 1. Go to the teaching workspace.

Reviewer: ```suggestion
1. Go to the teaching tab in the LearnOCaml Web UI (needs a teacher token).
```
(indeed, maybe workspace seems a bit ambiguous)

> 2. Click with the middle click on the exercise `x` in the list of exercise.

Reviewer: ```suggestion
2. Middle-click on the exercise `x` in the list of exercises.
```

3. Enter `foo` in the dialog box.

At this point, LearnOCaml opens a new window with the results.

> # References

Reviewer: ```suggestion
## References
```

For more details about how this classification internally works, have

> a look at [assak](https://github.com/nobrakal/asak), a library to

Reviewer: ```suggestion
a look at [asak](https://github.com/nobrakal/asak), a library to
```
